37.md

```diff
@@ -0,0 +1,30 @@
+NIP-37
+======
+
+Annotations
+-----------
+
+`draft` `optional`
+
+This NIP adds a new concept called "annotations" which allow authors to create privileged comments on their own notes.
+These are simply `kind:1` replies with an additional `annotation` tag.
+
+These can be used to indicate updates, edits, corrections, or retractions, without obscuring the content of the original
+note or forcing clients to handle multiple versions of the note. These MAY be applied to events of any kind.
+
+```js
+{
+  "kind": 1,
+  "pubkey": "<pubkey>",
+  "tags": [
+    ["e", "<event_id>", "<relay_hint>", "reply"],
```

Comment:
It would help to have another one letter tag that clients that opt-in to support this NIP could use to fetch annotations directly without getting mixed with regular replies from author.

Reply:
Wait. In the context of kind 1 threads (different from github replies for instance where one can't reply to a specific person's reply), aren't all OP (original poster) replies to themselves annotations already?

Reply:
No, they're just replies. I don't think the semantics would be the same. I also don't think a single letter tag is needed, how many times is someone going to reply to their own event?